pallets/did/src/benchmarking.rs

```diff
@@ -101,7 +104,11 @@ fn generate_base_did_call_operation<T: Config>(
 // encoded and which key takes the longest to verify and always use that.
```

Comment:
I would remove this comment by now then. We have found a workaround to that.

Reply:
6b058202dbd346633063c762ed0ef0f90a2898f1

Reply:
yeah didn't read this properly